### ISSUE TITLE
Adding game system symbaroum to the sources.json is not working

### ISSUE THREAD

**Issue by james3dwards** [League-of-Foundry-Developers/torch#76]
Hello.  
I need some guidance on what I might be doing wrong.  Can you point me to what I might be misunderstanding?

I followed the instructions to create an entry in the sources.json with game.systemid = symbaroum.  For some reason when I toggle the torch on/off, the radius is still at 40 dim/ 20 bright even after the custom setup.  Symbaroum uses Meters instead of feet so I want to switch the Dim/Bright to 12 and 6, plus enable some of the lighting effects.

Settings for the module under Core settings only has 3 check boxes to enabled and disable some things so not seeing much around configurable settings there.

On the token, I ensured the Torch in the inventory has a qty of 5 and it was equipped.  

I also have been reloading the game world between changes, and no success.

Lastly I tried removing all game systems and leaving just symbaroum and default in the .json.

Here is the section I created in the sources file.  

<img width="764" alt="image" src="https://github.com/League-of-Foundry-Developers/torch/assets/1271494/9512461d-6e97-47b2-9560-8b50e1a0e3be">

**Comment by lupestro** [comment]
A couple of things you could do - 

* First, for it not honoring specifically Torch - there’s a “feature” that’s confusing enough that I want to eliminate its effect on systems that have entries in the catalog. Try going into the config settings for Torch and change the name “Torch” there to just about anything else - “Flaming Banana”, maybe - and see if it starts respecting your Torch settings.
* Second, rather than changing the existing sources inside the module, make a separate JSON for your additional light sources and supply it through the config - that way it won’t be overwritten on the next update of the module. (If what you did was make a copy of sources.json, modify the copy, and then supply it as your additional light sources, that’ll work safely, too.)

**Comment by james3dwards** [comment]
Thank you for the help!

I switched the config and the torch icon changed to a "?"
Also tried changing my inventory to say "flaming banana" instead of torch.
Still no bueno.

{
  "symbaroum": {
    "system": "symbaroum",
    "topology": "standard",
    "quantity": "number",
    "sources": {
        "flaming banana": {
          "name": "flaming banana",
          "type": "equipment",
          "consumable": "false",
          "states": 2,
          "light": [
            { "bright": 12, "dim": 6, "angle": 360, "color":"#ff9329",
"alpha": 0.1,}
          ]
        }
    }
  },

**Comment by lupestro** [comment]
I'm sorry it took me so long to get back to you on this. I apparently responded in a confusing enough way that you completely misunderstood what I was saying. Let me try again:

> First, for it not honoring specifically Torch...Flaming Banana ...

What I meant was that in the config, where you have...
<img width="533" alt="image" src="https://github.com/League-of-Foundry-Developers/torch/assets/1447893/08f160a6-5e62-496a-b502-d5f4214e91dd">
... change _that_ to some nonsense like "flaming banana".

Then set up the symbaroum settings in a JSON the way you want them and point to the JSON file from:
<img width="541" alt="image" src="https://github.com/League-of-Foundry-Developers/torch/assets/1447893/dca71e02-cd3f-44ae-93cd-4b5c91d83270">

What you had up there looked correct, except don't include the ellipsis literally from the README. It was meant to indicate the author should "lather, rinse, repeat" with what they were doing.

For best results, before trying to use it, open your json file locally with Firefox to make sure the JSON syntax is correct. If there's a problem, Firefox will give you the line number where the first problem occurs. Using an editor like Visual Studio Code will also flag places where the syntax slides off the rails. This lets you progress a little faster because the "squigglies" tell you all the places it is hurting at once and as you fix things, they adjust accordingly.

Let me know if this gets you further. If you get a fairly comprehensive file for symbaroum that isn't too wildly customized, post it (or make a PR with it, if you're comfortable with that) and we could even include it in an upcoming release of the module.

**Comment by lupestro** [comment]
@james3dwards I just made some changes in 2.13.0 that should help with supplying additional light source JSON files. Let me know if this helps. I do really want to know if my responses have been helpful and what more you might need, so give me a holler. Especially let me know if I've addressed the intent of this ticket and can close it. 

In the interest of avoiding perpetually-open stale tickets, if I don't hear anything at all over the next two weeks, I may close it anyway, but you can always open a new one for whatever the situation has evolved to.

**Comment by james3dwards** [comment]
Yes, very helpful.
Thank you so much for the help.

You can close the ticket.  Thank you!
